README.md

```diff
@@ -30,6 +30,9 @@ For the complete list of changes made to schemachange check out the [CHANGELOG](
 1. [Authentication](#authentication)
    1. [Password Authentication](#password-authentication)
    1. [Private Key Authentication](#private-key-authentication)
+   1. [Oauth Authentication] (#oauth-authentication)
```

Comment:
remove the space between `] (` and the markdown reference will work.

Reply:
@MACKAT05

Reply:
Thanks!  I don't know how those spaces got in there since I had copy pasted one of the other elements to create them.

Reply:
This is some excellent work! Thank you for your contribution!